static/app/views/performance/landing/widgets/components/queryHandler.tsx

```diff
@@ -88,10 +89,15 @@ function QueryResultSaver<T extends WidgetDataConstraint>(
     results: any;
   } & QueryHandlerProps<T>
 ) {
+  const mepContext = useMEPContext();
   const {results, query} = props;
+
   const transformed = query.transform(props.queryProps, results, props.query);
 
   useEffect(() => {
+    const isMetricsData =
+      results?.seriesAdditionalInfo?.[props.queryProps.fields[0]]?.isMetricsData;
```

Comment:
Just checking the first field based on axis name, this should be sufficient for every event-stats perf widget, but will eventually cover this case w/ tests to ensure it works (since queryHandler data type is `any` still).